### ISSUE TITLE
Behavior of enter and spacebar for buttons

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/molecules-and-light#152]
From #143 @terracoda said

> Pressing and holding the Enter key plays continuous steps just like pressing and holding down the button with the mouse. However, pressing and holding the Space key does do the same thing. I have to release the Space key and then the items move a tiny bit and then stop.

>I think Enter and Space keys should behave the same. They should both do the same thing as pressing and holding the button down with the mouse.

> Adding: I was using Mac OSX 10.12.3 and Firefox 55.02 64-bit.

**Comment by jessegreenberg** [comment]
The difference is actually the standard behavior for HTML buttons.  Enter fires the button when pressed down (which is why you get press and hold behavior) and spacebar fires when the button is released. Doing something different might interfere with screen readers.

@terracoda does this seem correct to you, do you think it is worth deviating for PhET sims?

**Comment by terracoda** [comment]
Ok, that's great! Since that's the default behaviour, we should go with that. Thanks for checking. I was just operating on the idea that Enter and Space do the same thing. Thanks for clarifying that they don't **always** do the same thing!

**Comment by jessegreenberg** [comment]
Sounds good, I was surprised too @terracoda!  Closing.
